base/deprecated.jl

```diff
@@ -1076,4 +1077,6 @@ eval(Base.LinAlg, quote
     end
 end)
 
+Filesystem.stop_watching(stream::Filesystem._FDWatcher) = depwarn("stop_watching(::_FDWatcher) is removed")
```

Comment:
ZMQ was using `stop_watching`, though possibly not on this type. This is still not a useful warning, you haven't addressed that.